lib/adapter.js

```diff
@@ -100,9 +100,19 @@ var objectTrackingCallbackInterval =
 // Metrics API Use function
 function use(callback) {
 
+  exports.on('watchdogActivationCount', function(count) {
+    callback('watchdog.activations.count', count);
+  });
+
   // The callback needs to fire every 60 seconds with data from the last 60
   // seconds
   setInterval(function() {
+
+    if (appmetrics && appmetrics.watchdogActivationCount) {
```

Comment:
perhaps existence of the watchDogActivationCount property can be used to deterime if its supported? see comment above